lib/ansible/modules/cloud/docker/docker_image_info.py

```diff
@@ -22,6 +22,11 @@
 
 description:
      - Provide one or more image names, and the module will inspect each, returning an array of inspection results.
+     - Non-existing images will not be added to the result list. I.e. if you want to check whether an image exists
+       locally, you can call the module with this image, and check whether the result list is empty (image does not
+       exist) or has one element (the image exists locally).
+     - The module will not attempt to pull images from registries. Use M(docker_image) with I(source) set to O(pull)
```

Comment:
O() doesn't exist right? Typo?
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation

Reply:
Typo indeed! Should have been `C(...)`. Thanks for spotting!